### PR TITLE
Update hugo and node versions

### PR DESCRIPTION
**Pull request by dtzar**
Signed-off-by: David Tesar <david.tesar@microsoft.com>

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -142,9 +142,9 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.29"
+latest = "v1.28"
 
-version = "v1.29"
+version = "v1.28"
```
Comment (dipesh-rawat) [inline hugo.toml:147]: Why this switch from version v1.29 to v1.28?

Comment (dtzar) [reply hugo.toml]: Mistake from previous PR.  Fixed.

**Review by dipesh-rawat — COMMENTED** [review]
Checked out the [deploy preview](https://deploy-preview-44640--kubernetes-io-main-staging.netlify.app/) and it seems that these changes have affected the language selector dropdown in the navbar.

[Deploy Preview Website](https://deploy-preview-44640--kubernetes-io-main-staging.netlify.app/)

deploy-preview

**Review by Arhell — COMMENTED** [review]
Rebase please and  https://github.com/gohugoio/hugo/releases/tag/v0.121.2
Thanks

**Comment by Shubham82** [comment]
Thanks @dtzar
The preview looks good: https://deploy-preview-44640--kubernetes-io-main-staging.netlify.app/
Preview

**Comment by reylejano** [comment]
/area web-development

Looks like the language selector is working and website shows no negative impact
/approve

**Pull request merged**